workflow/controller/operator.go

```diff
@@ -1629,6 +1629,7 @@ func (woc *wfOperationCtx) inferFailedReason(pod *apiv1.Pod, tmpl *wfv1.Template
 		if t.Message != "" {
 			msg = fmt.Sprintf("%s: %s", msg, t.Message)
 		}
+		msg = fmt.Sprintf("%s in %s", msg, ctr.Name)
```

Comment:
I feel like `<name>: <message>` is easier to interpret than `<message> in <name>` for all of these